documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -178,12 +181,16 @@ Contains 32-bit Unix epoch of the date and time in UTC the record was written to
 | 24 | 2 | | Event type
 | 26 | 2 | | Number of strings
 | 28 | 2 | | Event category
-| 30 | 2 | | [yellow-background]*Unknown (Event flags)* +
-Actual usage unknown indicated as reserved
-| 32 | 4 | | [yellow-background]*Unknown (Closing record number)* +
-Actual usage unknown indicated as reserved
+| 30 | 2 | | *ReservedFlags (Event flags)* +
```

Comment:
Please leave as Unknown, fine that MS-EVEN claims that this is ReservedFlags or event flags, but its purpose is still unknown